src/java.base/share/classes/java/lang/Process.java

```diff
@@ -149,12 +169,93 @@ public abstract class Process {
     private Charset inputCharset;
     private BufferedReader errorReader;
     private Charset errorCharset;
+    private boolean closed;     // true if close() has been called
 
     /**
      * Default constructor for Process.
      */
     public Process() {}
 
+    /**
+     * Close all writer and reader streams and terminate the process.
+     * The streams are closed immediately and the process is terminated without waiting.
+     * This method is idempotent, if the process has already been closed
```

Comment:
I think make this `{@code Process}` as that is the handle to the process (the process has terminated, it hasn't been closed, if you see what I mean).